beets/mediafile.py

```diff
@@ -154,10 +154,12 @@ def _safe_cast(out_type, val):
             return int(val)
         else:
             # Process any other type as a string.
-            if not isinstance(val, six.string_types):
+            if isinstance(val, bytes):
+                val = val.decode('utf-8', 'ignore')
+            else:
```

Comment:
Cool; this looks right. Just to clarify what's going on, may I recommend `elif not isinstance(val, six.string_types)` here?